.github/workflows/create_release_assets_cross.yml

```diff
@@ -54,14 +64,30 @@ jobs:
 
       - name: Rename Release
         run: |
-          mkdir assets
+          mkdir -p assets
           FILENAME=topgrade-${{github.event.release.tag_name}}-${{matrix.target}}
           mv target/${{matrix.target}}/release/topgrade assets
           cd assets
           tar --format=ustar -czf $FILENAME.tar.gz topgrade
           rm topgrade
           ls .
 
+      - name: Build Debian-based system package without autoupdate feature
+        run: |
+          rm -rf target/${{matrix.target}}
+          cross build --release --features default --target ${{matrix.target}}
```

Comment:
Same here, the `default` feature is unnecessary

Reply:
Sure.